lib/data/path_data.dart

```diff
@@ -0,0 +1,31 @@
+const String PATH_MANIFEST = 'android/app/src/main/AndroidManifest.xml';
+const String PATH_MANIFEST_DEBUG = 'android/app/src/debug/AndroidManifest.xml';
+const String PATH_MANIFEST_PROFILE =
+    'android/app/src/profile/AndroidManifest.xml';
+
+const String PATH_ACTIVITY = 'android/app/src/main/';
+
+const String PATH_APP_LEVEL_GRADLE = 'android/app/build.gradle';
+const String PATH_TO_PLIST = 'ios/Runner/Info.plist';
+const String PATH_TO_PODFILE = 'ios/Podfile';
+const String PATH_AD_UNIT_ID = 'lib/ad_unit_ids/ad_unit_id.dart';
+const String PATH_MAIN = 'lib/main.dart';
+const String PATH_GRADLE_PROPERTIES = 'android/gradle.properties';
+
+// const String APPLICATION_ID = """\n        <meta-data
```

Comment:
@waleed-sofstica Please mention the usage of these variables. For now, it is not been used in any file. Are these useless?

Reply:
All of these variables are paths of files which are being configured. Each if these paths are being used

Reply:
The gradle.properties variable will be created while configuration if it doesn't already exists.
The main.dart file is used to place the initialization code of ads

Reply:
@waleed-sofstica the variable I commented is not being use anywhere. Give me a heads up to remove comments

Reply:
@Syed-Waleed-Shah The variable I commented is not being use anywhere. Give me e a heads up to remove comments